docs/making-your-own.md

```diff
@@ -29,8 +29,9 @@ The instructions below recommend `startingpoint` because it:
 - Is made to be modified and customized
 - Is more suitable for personal usage
 - Has an easier to edit configuration format, YAML, which also supports comments and inline documentation
+- Is very easy to customize in advanced ways via personal scripts, which "magically" run at build by [simply placing them in the appropriate folders](https://github.com/ublue-os/startingpoint/blob/template/scripts/README.md)
```

Comment:
```suggestion
- Advanced customization can be accomplished at built time by [placing scripts in the appropriate folders](https://github.com/ublue-os/startingpoint/blob/template/scripts/README.md)
```

Let's not add any more "simply" or "easy" loaded terms in the docs.

Reply:
I was just attempting to follow the existing tone, but agreed! Changed. :)